static_report/src/pages/SRTablesListPage.tsx

```diff
@@ -20,10 +13,13 @@ import { useReportStore } from '../utils/store';
 import { TableColumnSchemaList } from '../components/Tables/TableList/TableColumnSchemaList';
 import { useAmplitudeOnMount } from '../hooks/useAmplitudeOnMount';
 import { AMPLITUDE_EVENTS, SR_TYPE_LABEL } from '../utils/amplitudeEvents';
+import { CommonModal } from '../components/Common/CommonModal';
 
 type Props = { data: SaferSRSchema };
 
 export function SRTablesListPage({ data }: Props) {
+  const modal = useDisclosure();
```

Comment:
Same as above, I think the common modal should by default spread the disclosure's props inside the component itself, unless there's a case where that isn't desired?